keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -275,20 +275,48 @@ with a metric.
 
 There could be a large volume of requests for a specific resource which might result in the identified apiserver being unable to serve the proxied requests. This scenario should not occur too frequently, since resource types which have large request volume should not be added or removed during an upgrade -- that would cause other problems, too.
 
-TODO: security / cert stuff.
+We should ensure a definite way for the request flow to complete rather than continuing on indefinitely by proxying the request over and over again.
```

Comment:
```suggestion
We should ensure at most one proxy, rather than proxying the request over and over again (if the source apiserver has an incorrect understanding of what the destination apiserver can serve).
```

Reply:
Something happened here, I applied your commit but it got lost (I think I forgot to pull before pushing), I have made this change manually now though.